src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+const expensesService = require('../services/expenses.service');
+const userService = require('../services/users.service');
+const statusCodes = require('../vars/statusCodes');
+
+const getAll = (req, res) => {
+  const {
+    userId,
+    from,
+    to,
+    category,
```

Comment:
We don't have such field
<img width="587" alt="image" src="https://github.com/mate-academy/node_accounting-app-v2/assets/86789161/8dcb6216-d4a2-4e25-adc1-5e75e14d9330">